docs/source/development/getting-started.rst

```diff
@@ -12,13 +12,9 @@ To use the docker containter to develop:
 
 .. code-block:: bash
 
-    cd armada
-    export repo="https://github.com/<forked-repo>/armada.git"
-    export branch="<branch>"
+    docker build . -t armada/latest
```

Comment:
I added a third step to change to the directory of the repo - steps 1 and 2 say to fork and clone